main.js

```diff
@@ -291,6 +291,9 @@ app.on('activate', () => {
 // Emitted when all windows have been closed.
 app.on('window-all-closed', () => {
   logger.debug('window-all-closed');
+  if (process.platform === 'win32') {
+    app.quit();
```

Comment:
If the user has an encrypted wallet, this event will trigger after the user unlocks their wallet when the dialog prompt closes.

Reply:
@dwalintukan it is not related with wallet state. It is related with user cant sync data because qtumd is running

Reply:
`window-all-closed` will be trigger if a wallet is encrypted and the user enters the unlock passphrase. This will make the wallet unlock dialog close. And thus this will kill the app after the unlock it.

Reply:
@dwalintukan  `window-all-closed` will be triggered if the user closes all windows by clicking `x` on the top right window (https://github.com/electron/electron/blob/master/docs/api/app.md#event-window-all-closed).

Reply:
Yes but it also triggers when the wallet unlock dialog is closed (due to user clicking ok button). If this happens then the app will shutdown right after they unlock the wallet.

Reply:
A solution would be to start the startUI() func